internal/params/filters.go

```diff
@@ -135,6 +135,12 @@ var BaseFilters = []string{
 	"Directory.Packages.props",
 }
 
+var BaseExcludeFilters = []string{
+	"!.vs",
```

Comment:
should we use ! ?we are removing it by code no?

Reply:
Yes, the logic in the code removed it, but by documentation this is how we filter out a file:
https://checkmarx.com/resource/documents/en/34965-68643-scan.html#UUID-a0bb20d5-5182-3fb4-3da0-0e263344ffe7_id_scancreate-FileFilters:~:text=%2D%2Dfile%2Dfilter%20!mycompany.jar